cornice/pyramidhook.py

```diff
@@ -204,6 +204,11 @@ def register_service_views(config, service):
 
     routes = config.get_predlist('route')
     for predicate in routes.sorter.names:
+        # Do not let the custom predicates handle the Header Accept
+        # it is done in the fallback view
```

Comment:
I have trouble understanding why this change allows to use a callable for `accept`. I know I'm not extremely familiar with resources but this comment could be improved maybe? At least with a link to the related issue to help the reader.